scripts/distribute-lp-rewards/cli.js

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-param-reassign */
+// const assert = require('assert')
+const ethers = require('ethers')
+
+const { Contract, getDefaultProvider } = ethers
+const { bigNumberify, hexlify, Interface, id } = ethers.utils
+const fetch = require('node-fetch')
+const { Channel, Transaction, MerkleTree, splitSig } = require('adex-protocol-eth/js')
+const identityAbi = require('adex-protocol-eth/abi/Identity')
+
+const db = require('../../db')
+const cfg = require('../../cfg')
+const adapters = require('../../adapters')
+const {
+	getDistributionForPeriodWithMultiplier,
+	ZERO,
+	parseBalancerTransferEvents,
+	parseUniswapTransferEvents,
+	ADX_TOKEN,
+	DISTRIBUTION_IDENTITY,
+	FEE_TOKEN,
+	UNISWAP_ADX_ETH_ROUTER_ADDRESS,
+	BALANCER_ADX_YUSD_EXCHANGE_ADDRESS
+} = require('./lib')
+
+const coreAddr = cfg.ETHEREUM_CORE_ADDR
+
+const DISTRIBUTION_STARTS = new Date('2020-08-04T00:00:00.000Z')
+const DISTRIBUTION_ENDS = new Date('2020-12-31T00:00:00.000Z')
+
+const INCENTIVE_CHANNEL_OPEN_FEE = bigNumberify('1500000000000000000')
+const INCENTIVE_TO_DISTRIBUTE = bigNumberify('7010000000000000000000000')
+const DISTRIBUTION_PER_SECOND = bigNumberify('478927203065134100')
+
```

Comment:
Just to pin double check for distribution constants - start, end, amount, per second, etc.

Reply:
yes definitely, we haven't actually decided on the values yet though